src/sentry/static/sentry/app/views/settings/organizationRelays/dialogs/form.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import TextField from 'app/components/forms/textField';
+import TextareaField from 'app/components/forms/textareaField';
```

Comment:
Let's use the form components in `app/views/settings/components/form`

Reply:
can we please add  "this component is deprecated" inside `app/components/forms/* `?